2016-2017_Starstruck/Module Test Code/FileIO_Test/include/AutonRecorder.h

```diff
@@ -0,0 +1,87 @@
+/*
+ *  AutonRecorder.h
+ *
+ *  Created on: Feb 8, 2017
+ *      Author: Kirsten Meidlinger
+ */
+
+#ifndef AUTONRECORDER_H_
+#define AUTONRECORDER_H_
+
+/*
+ * This is the library for 750E's Auton Recorder. It reads joystick values and writes them to
+ *   a file in the cortex's memory, and is able to read the file again to replay it as an
+ *   autonomous program. It can work for any game.
+ *
+ * This file should be placed in include and included in main.h
+ */
+
+
+  typedef struct Frame {
+      int analog_main[4]; //ch1, ch2, ch3, ch4
+      bool digital_main[12]; //5U, 5D, 6U, 6D, 7U, 7D, 7L, 7R, 8U, 8D, 8L, 8R
```

Comment:
Might want to add macros for these to make it easier so we can do something like frame.digital_main[J_5U] to get values.